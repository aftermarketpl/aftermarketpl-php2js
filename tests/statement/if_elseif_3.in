$a = 4;
$b = 2;
$c = 3;
if($a == $b)
{
    $d = 1;
}
elseif($a == $c)
{
    $d = -1;
}
else
{
    $d = 0;
}
return $d;
