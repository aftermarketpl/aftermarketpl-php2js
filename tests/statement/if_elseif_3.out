var a, b, c, d;
a = 4;
b = 2;
c = 3;
if(a == b)
{
    d = 1;
}else if(a == c)
{
    d = -1;
}else
{
    d = 0;
}
return d;
